src/components/panel/panel.js

```diff
@@ -923,9 +1043,23 @@ MdPanelRef.prototype.open = function() {
   return this._$q(function(resolve, reject) {
     var done = self._done(resolve, self);
     var show = self._simpleBind(self.show, self);
+    var setOpened = function(response) {
```

Comment:
Why are you passing a response? I don't think it's necessary. This will always resolve, so we shouldn't care.